# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/validation/state/device_lost/destroy/cts.https.html.ini
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/validation/state/device_lost/destroy/cts.https.html.ini
--- a/testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/validation/state/device_lost/destroy/cts.https.html.ini	Fri Oct 03 18:01:30 2025 +0000
+++ b/testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/validation/state/device_lost/destroy/cts.https.html.ini	Fri Oct 03 18:08:29 2025 +0000
@@ -5439,37 +5439,39 @@
 
   [:format="rgba8sint";usageType="storage";usageCopy="dst";awaitLost=false]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and debug: [FAIL, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [PASS, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac": [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="storage";usageCopy="dst";awaitLost=true]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and debug: [FAIL, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [PASS, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac": [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="storage";usageCopy="none";awaitLost=false]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win": [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac": [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="storage";usageCopy="none";awaitLost=true]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win": [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac": [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="storage";usageCopy="src";awaitLost=false]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win": [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac": [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="storage";usageCopy="src";awaitLost=true]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win": [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac": [TIMEOUT, NOTRUN]
 
@@ -5487,51 +5489,60 @@
 
   [:format="rgba8sint";usageType="texture";usageCopy="dst";awaitLost=false]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and debug: [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and not debug: [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="texture";usageCopy="dst";awaitLost=true]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and debug: [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and not debug: [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="texture";usageCopy="none";awaitLost=false]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and debug: [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and not debug: [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="texture";usageCopy="none";awaitLost=true]
-    expected: [PASS, TIMEOUT, NOTRUN]
+    expected:
+      if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
+      if os == "linux": [PASS, TIMEOUT, NOTRUN]
+      if os == "mac": [PASS, TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="texture";usageCopy="src";awaitLost=false]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and debug: [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and not debug: [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="texture";usageCopy="src";awaitLost=true]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and debug: [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and not debug: [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="texture";usageCopy="src-dest";awaitLost=false]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win": [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac": [TIMEOUT, NOTRUN]
 
   [:format="rgba8sint";usageType="texture";usageCopy="src-dest";awaitLost=true]
     expected:
-      if os == "win": [PASS, TIMEOUT, NOTRUN]
+      if os == "win": [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and debug: [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and not debug: [TIMEOUT, NOTRUN]
@@ -5687,14 +5698,14 @@
   [:format="rgba8uint";usageType="render";usageCopy="dst";awaitLost=false]
     expected:
       if os == "win" and debug: [PASS, FAIL, TIMEOUT, NOTRUN]
-      if os == "win" and not debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac": [PASS, TIMEOUT, NOTRUN]
 
   [:format="rgba8uint";usageType="render";usageCopy="dst";awaitLost=true]
     expected:
       if os == "win" and debug: [PASS, FAIL, TIMEOUT, NOTRUN]
-      if os == "win" and not debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [PASS, TIMEOUT, NOTRUN]
       if os == "mac": [PASS, TIMEOUT, NOTRUN]
 
@@ -5705,16 +5716,32 @@
     expected: [PASS, TIMEOUT, NOTRUN]
 
   [:format="rgba8uint";usageType="render";usageCopy="src";awaitLost=false]
-    expected: [PASS, TIMEOUT, NOTRUN]
+    expected:
+      if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
+      if os == "linux": [PASS, TIMEOUT, NOTRUN]
+      if os == "mac": [PASS, TIMEOUT, NOTRUN]
 
   [:format="rgba8uint";usageType="render";usageCopy="src";awaitLost=true]
-    expected: [PASS, TIMEOUT, NOTRUN]
+    expected:
+      if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
+      if os == "linux": [PASS, TIMEOUT, NOTRUN]
+      if os == "mac": [PASS, TIMEOUT, NOTRUN]
 
   [:format="rgba8uint";usageType="render";usageCopy="src-dest";awaitLost=false]
-    expected: [PASS, TIMEOUT, NOTRUN]
+    expected:
+      if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
+      if os == "linux": [PASS, TIMEOUT, NOTRUN]
+      if os == "mac": [PASS, TIMEOUT, NOTRUN]
 
   [:format="rgba8uint";usageType="render";usageCopy="src-dest";awaitLost=true]
-    expected: [PASS, TIMEOUT, NOTRUN]
+    expected:
+      if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
+      if os == "win" and not debug: [FAIL, TIMEOUT, NOTRUN]
+      if os == "linux": [PASS, TIMEOUT, NOTRUN]
+      if os == "mac": [PASS, TIMEOUT, NOTRUN]
 
   [:format="rgba8uint";usageType="storage";usageCopy="dst";awaitLost=false]
     expected: [PASS, TIMEOUT, NOTRUN]